scripts/installer/curl_install_pypi/README.md

```diff
@@ -0,0 +1,34 @@
+# Curl Install Script Information
+
+The scripts in this directory are used for installing through curl and they point to the packages on PyPI.
+
+## Install or update promptflow
+
+curl https://promptflowartifact.blob.core.windows.net/linux-install-scripts/install | bash
+
+The script can also be downloaded and run locally. You may have to restart your shell in order for the changes to take effect.
+
+## Uninstall promptflow
+
+Uninstall the promptflow by directly deleting the files from the location chosen at the time of installation. The default install/executable location is the user's home directory ($HOME).
+
+1. Remove the installed CLI files.
+
+   ```bash
+   rm -r <install location>/lib/promptflow
+   rm <executable location>/bin/pf
```

Comment:
How do user know the value of install/executable location?

Reply:
may put the default path here

Reply:
user can input install/executable location when install by themselves. Default path will print in terminal also when install.  And the default path is ($HOME)